static-analysis/teammates-pmdMain.xml

```diff
@@ -6,14 +6,12 @@
     <description>
         This is the PMD additional ruleset for TEAMMATES production code.
     </description>
-
     <rule ref="rulesets/java/logging-java.xml">
         <!-- Not worth the added safety -->
         <exclude name="GuardLogStatementJavaUtil"/>        
-    </rule>
-
+    </rule>    
     <rule ref="rulesets/java/strictexception.xml/SignatureDeclareThrowsException"/>
-
+    <rule ref="rulesets/java/typeresolution.xml/LooseCoupling"/>
```

Comment:
Pls discard the rest of changes in `teammates-pmdMain.xml`. Only need to keep the changes for this line.